name_search_test.py

```diff
@@ -0,0 +1,49 @@
+import re
+import os
+import sys
+
+import name_search
+import scraper
+import pickle
+
+def search_many(src_dir, log_file_name):
+    """Writes all of the reports from src_dir into output_filename"""
+    pkl = True
+    if pkl:
+        with open('names_index.pickle','rb') as f:
+            charset, members_list, members_dict, members_index = \
+                    pickle.load(f)
+    else:
+        charset, members_list, members_dict, members_index = \
+                name_search.initialize()
+        with open('names_index.pickle','wb') as f:
+            pickle.dump((charset, members_list, members_dict, members_index), f)
+
+    count = 0
+    with open(log_file_name, 'w') as logfile:
+        for src_file_name in os.listdir(src_dir):
+            count += 1
+            print(count, src_file_name)
+            src_file_path = src_dir + src_file_name
+            for line in scraper.process_a_file(src_file_path):
+                t = [s[1:-1] for s in line.split(',')[:3]]
+                try:
+                    if (t[0].startswith('Hon')) or (t[0].startswith('Speaker')):
```

Comment:
There is an example in the files where it looks like there is a typo where "Hon" is "IIon".  I think we'll probably fix it in the scraper, but wanted to make sure it's noted here.